components/sst_importer/src/errors.rs

```diff
@@ -10,6 +10,45 @@ use grpcio::Error as GrpcError;
 use tokio_sync::oneshot::error::RecvError;
 use uuid::{parser::ParseError, BytesError};
 
+use crate::metrics::*;
+
+pub trait ErrorStatistics {
+    fn inc(&self);
+}
+
+impl ErrorStatistics for Error {
+    fn inc(&self) {
```

Comment:
Instead of a new trait, we can just define a function. It simplifies code.

Reply:
[Done](https://github.com/tikv/tikv/pull/6395/files#diff-71393c3184d770106b154c613c17a9aaR15)